Description: Makes the el-api optional since it's only used by BeanManager.
 This optimizes the transitive dependencies pulled when installing Maven.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/api/pom.xml
+++ b/api/pom.xml
@@ -140,6 +140,7 @@
 		<dependency>
 			<groupId>javax.el</groupId>
 			<artifactId>javax.el-api</artifactId>
+			<optional>true</optional>
 		</dependency>
 
 		<dependency>
